client/app/bundles/comments/components/CommentBox/CommentBox.scss

```diff
@@ -0,0 +1,21 @@
+:global {
```

Comment:
Let's add a couple comments on why `:global` is needed here and why we don't make this application level in that case. I'm guessing it's b/c of the bootstrap components have these classes, and we don't want to change those.